### PR TITLE
Slimebots drop the right baton.

### LINKED ISSUE #4870: Officer slimesky drops actual stun baton
#### Brief description of the issue
The Slimesky bot drops a sec issue stun baton when killed.
#### What you expected to happen
for it to drop a xenobio baton.
#### What actually happened
It did not drop a xenobio baton, it dropped a sec baton.
#### Steps to reproduce
Find a slimesky, kill it, enjoy your baton if you win.
#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.: Server revision: master - 2018-02-17
16295de3cfe858a753b29bb16c2503e31e04a92d 
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)
I am The Law

### PR DESCRIPTION
**Pull request by MisterLayne**
As opposed to Slimebots dropping the left baton.
Fixes #4870

### CODE REVIEW

**Comment by Anewbe** [comment]
I feel like this could be done by assigning a var to secbots for their appropriate baton, then passing that var into the explode() proc.

**Comment by MisterLayne** [comment]
@Anewbe would you prefer it be done that way? I suppose I could assign a baton to all types of bots like weapon, and shove that on the ED as well?

**Comment by Anewbe** [comment]
Is the ED a child of the same parent?

**Comment by MisterLayne** [comment]
@Anewbe indeed it is.

File: code/modules/mob/living/bot/ed209bot.dm [hunk 1]
```diff
@@ -30,7 +32,7 @@
 
 	new /obj/item/weapon/secbot_assembly/ed209_assembly(Tsec)
 
-	var/obj/item/weapon/gun/energy/taser/G = new /obj/item/weapon/gun/energy/taser(Tsec)
+	var/obj/item/weapon/gun/energy/taser/G = new used_weapon(Tsec)
```
Comment (MisterLayne) [inline code/modules/mob/living/bot/ed209bot.dm]: Not sure if this will cause problems?

Comment (Anewbe) [reply code/modules/mob/living/bot/ed209bot.dm]: I'm not sure either, so... We'll find out!

**Pull request merged**